### PR TITLE
Fix response filters

### PR DESCRIPTION
**Pull request by stnguyen90**
## What does this PR do?

Fix:

1. Document $collectionId to $collection
2. Fix empty DateTimes to 0

## Test Plan

Unit tests

## Related PRs and Issues

None

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Comment by stnguyen90** [comment]
Passing tests:

image

File: tests/unit/Utopia/Response/Filters/V15Test.php [hunk 1]
```diff
@@ -1077,6 +1102,18 @@ public function sessionProvider(): array
                     'providerAccessTokenExpiry' => 1592981250,
                 ],
             ],
+            'empty values' => [
+                [
+                    '$createdAt' => '',
+                    'expire' => '',
```
Comment (abnegate) [inline tests/unit/Utopia/Response/Filters/V15Test.php:1108]: What scenario would cause these to be empty?

Comment (stnguyen90) [reply tests/unit/Utopia/Response/Filters/V15Test.php:1108]: session.proverAccessTokenExpiry can be an empty string.

Comment (abnegate) [reply tests/unit/Utopia/Response/Filters/V15Test.php:1108]: But `$createdAt` and `expire` can not, right? Let's update the values to reflect that for clarity

Comment (stnguyen90) [reply tests/unit/Utopia/Response/Filters/V15Test.php]: Updated

**Review by abnegate — APPROVED** [review]

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**